java/util/src/main/java/com/google/protobuf/util/JsonFormat.java

```diff
@@ -294,14 +295,17 @@ public static Parser parser() {
   public static class Parser {
     private final TypeRegistry registry;
     private final boolean ignoringUnknownFields;
+    private final boolean usingUnrecognizedEnumValue;
```

Comment:
The difference between ignore field and this one is that the unknown enum
isn't being ignored. We're using that information to generate the
unrecognized enum.